packages/calypso-analytics/README.md

```diff
@@ -0,0 +1,3 @@
+# Calypso Analytics Package
+
+## Usage
```

Comment:
Maybe you got throw in a brief word about how this only includes track events. And a link to `client/lib/analytics/docs/tracks.md`?